cranelift/codegen/src/isa/aarch64/lower.rs

```diff
@@ -1243,6 +1243,150 @@ pub(crate) fn maybe_input_insn_via_conv<C: LowerCtx<I = Inst>>(
     None
 }
 
+pub(crate) fn match_vec_long_mul<C: LowerCtx<I = Inst>>(
```

Comment:
Can we add a doc-comment here describing the return tuple? E.g. it's not clear to me reading at this point what the `bool` denotes, until I notice the low/high pattern below in the implementation.